deploy/templates/nstemplatetiers/base/ns_stage.yaml

```diff
@@ -7,22 +7,22 @@ objects:
   kind: Namespace
   metadata:
     annotations:
-      openshift.io/description: ${USERNAME}-stage
-      openshift.io/display-name: ${USERNAME}-stage
-      openshift.io/requester: ${USERNAME}
+      openshift.io/description: ${SPACE_NAME}-stage
+      openshift.io/display-name: ${SPACE_NAME}-stage
+      openshift.io/requester: ${SPACE_NAME}
```

Comment:
Technically it's supposed to be a user but I guess it's fine to use the space name here. Especially for the cases when we don't have a user when creating a space.

Reply:
Right, maybe at the moment `SPACE_NAME` makes more sense for the `appstudio-env` tier. 
But still having different parameter names based on the `tier` I don't think it would be a good idea. 

Thanks for review and approval!